src/themes/nautilus/index.js

```diff
@@ -211,6 +220,15 @@ export const theme = {
           strokeWidth: 1.125,
         },
       },
+
+      // TODO: implement these!
+      Tag: {
```

Comment:
We should probably use a "z" here. And these should be added to the theme prop-types. Maybe I'll organise some tests to verify the prop-types are up-to-date...

Reply:
....Tagz?

Reply:
I have no idea where that came from? Oh, I know! Copy/pasted from the wrong line.

Reply:
(Where are the theme prop-types stored?)

Reply:
https://github.com/octopusthink/nautilus/blob/master/src/themes/propTypes.js

But don't sweat it for now. I just mentioned it for awareness/posterity really.

Reply:
I was definitely not aware that file even existed. 😮 

Probably worth doing a release focussed on themeing in the near-ish future (maybe 0.6 or 0.7) and looking into these sorts of things in a bit more depth—there are a number of things I think we can improve about the structure of the themeing itself, and I think they're probably worth tackling once we have Foundation and Function a bit more fleshed out.